File: Python/1 - Iniciante/1132.py
# coding: UTF-8
# Funcional no URI

a = int(input())
b = int(input())
soma = int(0)
if a > b:
    ran = a - b
    menor = b - 1
    maior = a
else:
    ran = b - a
    menor = a - 1
    maior = b

while menor < maior:
    menor += 1
    if menor % 13 != 0:
        soma += menor

print(soma)

File: go/1132.go
// Funcional no beecrowd
package main

import (
	"fmt"
)

func main() {
	var x int
	var y int
	soma := 0
	maior := 0
	menor := 0

	fmt.Scan(&x)
	fmt.Scan(&y)

	if x > y {
		maior = x
		menor = y
	} else {
		maior = y
		menor = x
	}

	for i := menor; i <= maior; i++ {
		if i%13 != 0 {
			soma += i
		}
	}

	fmt.Println(soma)
}
